simde/arm/neon/ceq.h

```diff
@@ -21,7 +21,7 @@
  * SOFTWARE.
  *
  * Copyright:
- *   2020      Evan Nemerson <evan@nemerson.com>
+ *   2023      Yi-Yen Chung <eric681@andestech.com> (Copyright owned by Andes Technology)
```

Comment:
```suggestion
*   2020      Evan Nemerson <evan@nemerson.com>
```
Evan wrote the first version of this file..